src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -1092,6 +1109,47 @@ export default {
 
       return (this.disableChanges || item.isCorrection || item.isFutureEffectiveIa || item.isColinFiling ||
         disableThisIaCorrection)
+    },
+
+    /**
+     * Fetches a filing by its id.
+     * @param businessId The business identifier
+     * @param filingId The filing identifier
+     * @returns a promise to return the filing of the specified type
```

Comment:
There is no reference to type here. Please update this comment.

Reply:
Updated in next commit!